compas/reverse-proxy/nginx.conf

```diff
@@ -66,6 +63,24 @@ http {
        proxy_redirect off;
      }
 
+     # Forwarding to the SCL Validator Service container (websockets).
+     location /compas-scl-data-service/scl-ws/ {
+       include /etc/nginx/include/authenticate.include;
+
+       proxy_http_version 1.1;
+       proxy_set_header Upgrade           $http_upgrade;
+       proxy_set_header Connection        "Upgrade";
+       proxy_set_header Host              $host;
+
+       proxy_set_header X-Real-IP         $remote_addr;
+       proxy_set_header X-Forwarded-For   $proxy_add_x_forwarded_for;
+       proxy_set_header X-Forwarded-Proto $scheme;
+       proxy_set_header X-Forwarded-Port  $server_port;
+       proxy_set_header Authorization     "Bearer ${access_token}";
+
+       proxy_pass http://scl-data-service:8080/compas-scl-data-service/scl-ws/;
```

Comment:
moet dit geen ws:// zijn?

Reply:
This is the way I already found for the compas-scl-validator-service in the past in Nginx.
It works, but maybe there is a better way.